src/routes/entry/[slug].svelte

```diff
@@ -47,9 +47,16 @@
 {:else if entry.visibility === 'hidden'}
 	<p class="visibility-warning">Hidden</p>
 {/if}
+
 <h1>{entry.title}</h1>
 <article>{@html markdown.render(entry.content)}</article>
 
+{#if entry.tags}
+	{#each entry.tags as tag}
+		<p class="tag">{tag}</p>
```

Comment:
```suggestion
		<span class="tag">{tag}</span>
```
This doesn't *really* matter but I believe span is a bit more fitting here.